src/torchcodec/_core/CMakeLists.txt

```diff
@@ -183,7 +223,9 @@ function(make_torchcodec_libraries
     install(
         TARGETS ${all_libraries}
         LIBRARY DESTINATION ${CMAKE_INSTALL_PREFIX}
+        RUNTIME DESTINATION ${CMAKE_INSTALL_PREFIX}  # For Windows DLLs
```

Comment:
Saw this in audio: https://github.com/pytorch/audio/blob/9b57c7bd5ad4db093c5bb31c802df9f04d933ac9/cmake/TorchAudioHelper.cmake#L29